configure.ac

```diff
@@ -89,7 +89,7 @@ AC_SUBST([ENABLE_CLANG_PARSER])
 
 PKG_CHECK_MODULES([LIBFFI], [libffi >= 3.2])
 PKG_CHECK_MODULES([BABELTRACE2], [babeltrace2 >= 2.0])
-PKG_CHECK_MODULES([LTTNG_UST], [lttng-ust >= 2.10])
+PKG_CHECK_MODULES([LTTNG_UST], [lttng-ust >= 2.12.8])
```

Comment:
Why?  We are using new LTTNG UST API?

Reply:
We need this for getting `__attribute__((constructor))` working.